config/consul.json

```diff
@@ -1,10 +1,9 @@
 {
 	"data_dir": "/data",
-	"ui_dir": "/ui",
 	"client_addr": "0.0.0.0",
 	"ports": {
 		"dns": 8600
 	},
-	"recursor": "10.200.16.2",
+	"recursors": ["10.200.16.2"],
```

Comment:
Is having 1 recursion safe enough?

Reply:
Added some more in `0ccdec4d801008292db744d5cb09065409ec099f`.